packages/gestalt/src/DropdownItem.jsdom.test.js

```diff
@@ -20,7 +20,7 @@ describe('Dropdown.Item', () => {
   test('adds badge', () => {
     render(
       <Dropdown.Item
-        badgeText="Beta Badge"
+        badge={{ text: 'Beta Badge' }}
```

Comment:
Should badges use sentence case? (I have no idea)

Reply:
They probably should, all our docs do, just haven't worried about test values